app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,879 @@
+    {
+        title: 'Spiderman',
+        year: 2002,
+        reviews: [
+            { author: 'Bob', text: 'Great movie!' },
+            { author: 'Alice', text: 'Loved it!' }
+        ]
+    }
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1')  // Your API Endpoint
+    .setProject('5df5acd0d48c2');                // Your project ID
+
+final databases = Databases(client);
+
+await databases.createDocument(
+  databaseId: 'marvel',
+  collectionId: 'movies',
+  documentId: ID.unique(),
+  data: {
+    'title': 'Spiderman',
+    'year': 2002,
+    'reviews': [
+      { 'author': 'Bob', 'text': 'Great movie!' },
+      { 'author': 'Alice', 'text': 'Loved it!' }
+    ]
+  },
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Apple</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+let databases = Database(client: client)
+
+databases.createDocument(
+    databaseId: "marvel",
+    collectionId: "movies",
+    documentId: ID.unique(),
+    data: [
+        "title": "Spiderman",
+        "year": 2002,
+        "reviews": [
+            [ "author": "Bob", "text": "Great movie!" ],
+            [ "author": "Alice", "text": "Loved it!" ]
+        ]
+    ]
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Database
+import io.appwrite.ID
+
+val client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+    .setProject("5df5acd0d48c2")                // Your project ID
+
+val databases = Database(client)
+
+databases.createDocument(
+    databaseId = "marvel",
+    collectionId = "movies",
+    documentId = ID.unique(),
+    data = mapOf(
+        "title" to "Spiderman",
+        "year" to 2002,
+        "reviews" to listOf(
+            mapOf("author" to "Bob", "text" to "Great movie!"),
+            mapOf("author" to "Alice", "text" to "Loved it!")
+        )
+    )
+)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<p>If a nested document is included and <b>no conflicting document ID</b> exists, the child document will be <b>created</b>.</p>
+<p>If a nested document is included and the <b>document ID already exists</b>, the child document will be <b>updated</b>.</p>
```

Comment:
Let's make it more explicit whether we're talking about the parent document's ID or the child document's ID.

Also coming to the topic of child document IDs, do we need to pass them when creating the nested documents ? The examples leave the reader with this ambiguity. 

And what is the behaviour if a child document ID already exists? Is the existing one updated ? Or is an error thrown ?

Reply:
>Let's make it more explicit whether we're talking about the parent document's ID or the child document's ID.

Sounds good.

> Also coming to the topic of child document IDs, do we need to pass them when creating the nested documents ? The examples leave the reader with this ambiguity.

We don't need to pass them in.

>And what is the behaviour if a child document ID already exists? Is the existing one updated ? Or is an error thrown ?

Error is thrown.